contingent_plan_executor/hovor/__init__.py

```diff
@@ -20,29 +20,28 @@ def setup(sqlalchemy_db_uri):
     db.init_app(app)
     return app, db
 
-# also write the plan configuration data to the volume
-if len(sys.argv) > 1:
-    output_files_path = sys.argv[1]
-else:
-    raise ValueError("Please provide the directory to your plan4dial output files as a system argument.")
-    # output_files_path = "local_data/updated_gold_standard_bot" # ONLY for local debugging
+def setupapp():
+    # also write the plan configuration data to the volume
+    if len(sys.argv) > 1:
+        output_files_path = sys.argv[1]
+    else:
+        raise ValueError("Please provide the directory to your plan4dial output files as a system argument.")
+        # output_files_path = "local_data/updated_gold_standard_bot" # ONLY for local debugging
 
-# save the output files path name
-with open("out_path.txt", "w") as f:
-    f.write(output_files_path)
+    # save the output files path name
```

Comment:
This a mix of tabs/spaces?

Reply:
I just tabbed everything once (and I just went in to check to make sure the spaces are even). Not sure why it's showing up funky :P

Reply:
But when you select those lines is it using spaces? Sometimes tabs can creep in

Reply:
Don't think I see a difference in my editor - I went and re-tabbed everything in my IDE and it didn't show any file changes. But I just ran black on it, hope that helps!